src/proto/proto_common.cpp

```diff
@@ -792,10 +796,10 @@ void print_help(lbann_comm *comm)
 
   std::cerr <<
        "General usage: you need to specify three prototext files, e.g:\n"
-       "  srun -n# proto --model=<string> --optimizer=<string> --reader=<string>\n"
+       "  srun -n# proto --model=<string> --optimizer=<string> --reader=<string> --metadata=<string>\n"
        "\n"
        "  However, if you are re-running an experiment from a previously saved\n"
-       "  file, you only need to specify --model=<string>\n"
+       "  file, you only need to specify --prototext=<string>\n"
```

Comment:
I think "prototext" is kind of too general and does not give much hint of what it is for.